tests/steps/cluster.py

```diff
@@ -467,6 +467,13 @@ def step_container_is_replica(context, replica_name, primary_name):
     return assert_container_is_replica(context, replica_name, primary_name)
 
 
+@then('container "(?P<replica_name>[a-zA-Z0-9_-]+)" is a replica of container "(?P<primary_name>[a-zA-Z0-9_-]+)" and streaming')
+@helpers.retry_on_assert
+def step_container_is_replica_and_streaming(context, replica_name, primary_name):
+    return assert_container_is_replica(context, replica_name, primary_name) \
+        and step_container_is_in_quorum_group_and_streaming(context, replica_name)
```

Comment:
No need to construct this long and, since both `assert_container_is_replica` and `step_container_is_in_quorum_group_and_streaming` are throwing AssertionError and do not return anything:
```suggestion
    assert_container_is_replica(context, replica_name, primary_name)
    step_container_is_in_quorum_group_and_streaming(context, replica_name)
```